### PR TITLE
ResNet 9||CIFAR10||davidcpage

### PR DESCRIPTION
**Pull request by davidcpage**
CIFAR10 training in 74 seconds using a custom ResNet 9

### CODE REVIEW

File: CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "v1.0",
+    "author": "davidcpage",
```
Comment (codyaustun) [inline CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json]: Is this how you want your name to appear on the website? "David Page" or "Myrtle: David Page" might be more human readable.

File: CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "authorEmail": "david@myrtlesoftware.com",
+    "framework": "pytorch 0.4.0",
+    "codeURL": "https://github.com/davidcpage/cifar10-fast/tree/d31ad8d393dd75147b65f261dbf78670a97e48a8", 
+    "model": "Custom ResNet 9",
+    "hardware": "V100 (AWS p3.2xlarge)",
+    "costPerHour": 3.060,
+    "timestamp": "2018-11-06",
+    "misc": {
```
Comment (codyaustun) [inline CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json:11]: For convenience, can you add optimizer details and hyperparameters (e.g. Nesterov, weight decay, batch size, and learning rate) here? As an example, see this [submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/ba0cd1d4bd8b1eb3aea5d47e67207cc03afeddf7/ImageNet/train/google_resnet50_tpu_tensorflow111.json#L18).

**Review by codyaustun — CHANGES_REQUESTED** [review]
Thanks for the submission. It looks good overall, but please add more details under the `misc` field.

File: CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json (hunk 1 above)
Comment (davidcpage) [reply CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json]: Thanks I've updated this.

File: CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json (hunk 2 above)
Comment (davidcpage) [reply CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json:11]: I've added optimizer details and parameters. Thanks for the speedy review!

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Looks good to me! Thanks!
